data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -69,8 +69,8 @@
       % if ($check_var->('SYSTEM_ROLE', 'kvm')) {
         % if ($check_var->('AMD', '1')) {
           % if ($check_var->('VIRT_SEV_ES_GUEST_INSTALL', '1')) {
-            <!-- Do not add mem_encrypt=on option on 15-SP6 KVM host due to bsc#1224107 -->
-            % if ($check_var->('VERSION', '15-SP6')) { 
+            <!-- Do not add mem_encrypt=on on kernel command line for amd-zen3-gpu-sut1 which is the only machine affected by bsc#1224107 -->
+            % if ($check_var->('WORKER_CLASS', '64bit-ipmi,64bit-ipmi-large-mem,64bit-ipmi-amd,64bit-ipmi-amd-zen3,64bit-ipmi-vgpu,amd-zen3-gpu-sut1,zone-cc,region-prg,datacenter-dc7,location-prg2e,grenache-1,cpu-ppc64le')) { 
```

Comment:
When adding/updating the worker classes of the machine, it may easily break this check_var. Suggest to loose the check, eg `if ($get_var->('WORKER_CLASS') =~ /64bit-ipmi-amd-zen3/)`, if this is enough to differentiate which machine can not have the option.

Reply:
Done